### PR TITLE
ScreenManager: redraw current FunctionBar on resize

### PR DESCRIPTION
**Pull request by cgzones**
Closes: #402

### CODE REVIEW

**Review by BenBE — APPROVED** [review]

**Comment by BenBE** [comment]
Resizing in the DisplayOptions panel fixed with this patch applied.

**Comment by cgzones** [comment]
Currently this breaks the FunctionBar when using Filter or Search mode.
I'll rethink the whole FunctionBar drawing logic...

File: CommandScreen.c [hunk 1]
```diff
@@ -17,7 +17,7 @@ static void CommandScreen_scan(InfoScreen* this) {
    Panel_prune(panel);
 
    const char* p = Process_getCommand(this->process);
-   char* line = xMalloc(COLS + 1);
+   char line[COLS + 1];
    int line_offset = 0, last_spc = -1, len;
    for (; *p != '\0'; p++, line_offset++) {
       line[line_offset] = *p;
```
Comment (BenBE) [inline CommandScreen.c:23]: Add extra `assert(line_offset < COLS + 1)` as cautionary measure when converting this to a stack buffer.

**Comment by BenBE** [comment]
Drawn this way the function bar looks kinda weird:
image
Also with large fonts the two options for Del and F10 are pushed out of the display area.

**Pull request merged**